build.sbt

```diff
@@ -566,6 +568,23 @@ lazy val scioRepl: Project = Project(
   scioExtra
 )
 
+lazy val scioJmh: Project = Project(
+  "scio-jmh",
+  file("scio-jmh")
+).settings(
+  commonSettings,
+  noPublishSettings,
```

Comment:
To keep it consistent with other project:
 * `commonSettings ++ noPublishSettings`
 * add `description`

Reply:
Fixed